store/tikv/region_cache.go

```diff
@@ -1173,6 +1173,7 @@ type Store struct {
 	resolveMutex sync.Mutex   // protect pd from concurrent init requests
 	fail         uint32       // store fail count, see RegionStore.storeFails
 	storeType    kv.StoreType // type of the store
+	tokenCount   int64        // used store token count
```

Comment:
Prefer to use `github.com/uber-go/atomic`

Reply:
Addressed.